src/modules/AdvancedPaste/AdvancedPaste/Helpers/AICompletionsHelper.cs

```diff
@@ -127,6 +129,16 @@ public AICompletionsResponse AIFormatString(string inputInstructions, string inp
                 apiRequestStatus = -1;
             }
 
+            int promptTokens = -1;
+            int completionTokens = -1;
+            if (rawAIResponse != null)
+            {
+                promptTokens = rawAIResponse.Value.Usage.PromptTokens;
+                completionTokens = rawAIResponse.Value.Usage.CompletionTokens;
+            }
+
+            PowerToysTelemetry.Log.WriteEvent(new Telemetry.AdvancedPasteGenerateCustomFormatEvent(promptTokens, completionTokens, "gpt-3.5-turbo-instruct"));
```

Comment:
uplevel "gpt-3.5-turbo-instruct" into shared var.

it is used at a min on `DeploymentName = "gpt-3.5-turbo-instruct",` higher up in code

Reply:
Do we need to send the event even when it fails? Wondering if -1 values won't just skew the data unless you can filter those out.

Reply:
Fixed both of these: made a variable for model name and put the telemetry event in the try block